src/routes/CommunitySettings/ImportExportSettingsTab/ImportExportSettingsTab.js

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import Button from 'components/Button'
+import './ImportExportSettingsTab.scss'
+
+export default class ImportExportSettingsTab extends Component {
```

Comment:
I like functional components for new stuff.

Reply:
ok ok, will get on that moving forward :)